src/modules/github/components/PullReviewForm.tsx

```diff
@@ -85,6 +85,38 @@ export const PullReviewForm: FC<PullReviewFormProps> = ({
     })
   }
 
+  const handleApproveClick = () => {
+    console.log('approve click')
+
+    const checkedPullsInfo = getCheckedPullsInfo(repoHasCheckArray)
```

Comment:
this is the reason i separate `getCheckedPullsInfo` function so you can use it for "comment", "approve", and "request change".

Reply:
uhm, okay!